nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -1275,11 +1280,10 @@ public void run() {
                         gzip ? ".gz" : ""));
             }
             log.info("Dumping connect report in " + outputpath);
-            OutputStream output = Files.newOutputStream(outputpath, StandardOpenOption.TRUNCATE_EXISTING, StandardOpenOption.CREATE);
-            if (gzip) {
-                output = new GZIPOutputStream(output, 512, true);
+            try (OutputStream output = Files.newOutputStream(outputpath, StandardOpenOption.TRUNCATE_EXISTING, StandardOpenOption.CREATE)) {
+                commandSucceeded = launcher.dumpConnectReport(gzip ? new GZIPOutputStream(output) : output,
```

Comment:
`GZIPOutputStream` should be closed, `GZIPOutputStream#close` performs additional operations. Extract to a method `OutputStream` instantiation should do the work.